src/store/RootState.ts

```diff
@@ -5,11 +5,18 @@ export interface SubmitError {
 	message?: string;
 }
 
+interface PerFieldError {
+	messageKey: string;
+}
+
 export default interface RootState {
 	lemma: string;
 	language: SearchedItemOption | null;
 	languageCodeFromLanguageItem: string | undefined | null | false;
 	lexicalCategory: SearchedItemOption | null;
 	spellingVariant: string;
 	globalErrors: SubmitError[];
+	perFieldErrors: {
+		lemmaErrors: PerFieldError[];
```

Comment:
Do these errors get cleared anywhere?

Reply:
Huh, I was so enamored with _adding_ those errors, that I fully forgot about clearing them! Will add the functionality right away!

Reply:
Added it. Not sure how to best test it though :thinking:

Reply:
I guess that feels like an integration test? But we could also do that at the end of the chain, for all inputs at once (errors are added on submit and cleared one by one as input is added).